pkg/frontend/nvme_subsystem.go

```diff
@@ -80,6 +80,7 @@ func (s *Server) CreateNvmeSubsystem(_ context.Context, in *pb.CreateNvmeSubsyst
 		params := spdk.NvmfSubsystemAddHostParams{
 			Nqn:  in.NvmeSubsystem.Spec.Nqn,
 			Host: in.NvmeSubsystem.Spec.Hostnqn,
+			Psk:  "/tmp/opikey.txt",
```

Comment:
I think to merge like this and then move PSK file save into utils and use it here... WDYT @artek-koltun ?

Reply:
agree